pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -1492,6 +1492,125 @@ func TestSchedulerCache_UpdateSnapshot(t *testing.T) {
 	}
 }
 
+func TestSchedulerCache_updateNodeInfoSnapshotList(t *testing.T) {
```

Comment:
pedantic nit: Could you instead add the case to `TestSchedulerCache_UpdateSnapshot`?

General recommendation in go is to test the public interface (`UpdateSnapshot` function in this case). Also it will make the patch smaller.

Reply:
On the other hand one would argue that this test will be easier to adapt to test the `toList` function that we plan to have as a permanent fix.